Cargo.toml

```diff
@@ -4,19 +4,19 @@ version = "0.1.0"
 edition = "2021"
 
 [dependencies]
-geozero = { git = "https://github.com/bitner/geozero.git", rev = "45828d7" }
-boon = "0.6.0"
-lazy_static = "1.5.0"
-pest = "2.7.11"
-pest_derive = "2.7.11"
-serde = "1.0.204"
-serde_derive = "1.0.204"
-thiserror = "1.0.63"
-serde_json = { version = "1.0.120", features = ["preserve_order"] }
+geozero = "0.14"
+boon = "0.6"
+lazy_static = "1"
+pest = "2"
```

Comment:
I think changing `2.7.x` to `2` is especially risky, because in theory there are _new features_ in the minor releases from `2` to `2.7` (and not just bug fixes from `2.7` to `2.7.11`). And so there's no way to tell in isolation whether we use any of the features added in `2.2` or `2.5` that didn't exist in `2.0.0`.